src/components/AllReviews.jsx

```diff
@@ -0,0 +1,32 @@
+import { useEffect } from "react";
+import { useState } from "react";
+import ReviewCard from "./ReviewCard"
+
+export default function AllReviews() {
+    const[reviews, setReviews] = useState([]);
+
+
+    useEffect(()=>{
+        fetch("https://bencgames.herokuapp.com/api/reviews")
+        .then((res)=>{
+            return res.json()
+        }).then ((response)=>{
+            console.log(response)
+            setReviews(response.reviews)
+        })
+    },[])
+    return (
+        <section className="AllReviews">
+            {reviews.map(({title, owner, votes, review_id})=>{
```

Comment:
I might have been tempted to use the isLoading pattern here but its working all good :)